app/helpers/form_helper.rb

```diff
@@ -304,16 +304,24 @@ def help_inline(inline, error)
   def add_label options, f, attr
     label_size = options.delete(:label_size) || "col-md-2"
     required_mark = check_required(options, f, attr)
-    label = options[:label] == :none ? '' : options.delete(:label)
-    label ||= ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
-        s_(clazz.gettext_translation_for_attribute_name attr)) if f
-    label = label.present? ? label_tag(attr, "#{label}#{required_mark}", :class => label_size + " control-label") : ''
+    if options[:label] == :none
+      label = ''.html_safe
+    else
+      label = ''.html_safe + options.delete(:label)
+      label += ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
+        s_(clazz.gettext_translation_for_attribute_name attr).html_safe) if f && label.empty?
+    end
+
+    if options[:label_help].present?
+      label += ' '.html_safe + popover("", options[:label_help], options[:label_help_options] || {})
```

Comment:
it is, or I need to write it as `label = label + ' '.html_safe + popover("", options[:label_help], options[:label_help_options] || {})`